aperturedb/OM/Entity.py

```diff
@@ -0,0 +1,74 @@
+from typing import ByteString, List, Optional, Dict
+from aperturedb import Connector
+
+
+class Entity:
```

Comment:
this is a confusing name, because we have one type of object on ApertureDB that is called "Entity"

Reply:
Yes, I was thinking the same. I am trying to think alternative, Object and unit comes to mind. I think will rename it to Unit. Do you have any specific word in mind?

Reply:
"Thingy", "Thing", "Sometning", "ADObject", "Substance", "Root", "Stuff".

Reply:
FWIW, I vote for something like `Object` or `BaseObject` since this is the Object Mapper.